compose/case-document-am.yml

```diff
@@ -5,16 +5,13 @@ services:
   ccd-case-document-am-api:
     image: "${CCD_CASE_DOCUMENT_AM_API_USE_LOCAL-hmctspublic.azurecr.io/}ccd/case-document-am-api:${CCD_CASE_DOCUMENT_AM_API_TAG:-latest}"
     environment:
-      CCD_DOCUMENT_API_IDAM_KEY: "${IDAM_KEY_CASE_DOCUMENT}"
+      CASE_DOCUMENT_AM_API_S2S_SECRET: "${IDAM_KEY_CASE_DOCUMENT}"
       CASE_DOCUMENT_S2S_AUTHORISED_SERVICES: ccd_case_document_am_api,ccd_gw,xui_webapp,ccd_data,bulk_scan_processor
       IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
       IDAM_OIDC_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
-      S2S_URL: http://service-auth-provider-api:8080
-      IDAM_USER_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      S2S_URL: http://service-auth-provider-api:8080     
       DOCUMENT_STORE_URL: http://dm-store:8080
```

Comment:
Should this be `DM_STORE_BASE_URL`?

Reply:
yes fixed in 0fe91dfcbcdb903672b88bb62ed98f6000ecc758